File: bl-central-server/bl-sensor-data-collector/src/main/java/org/jesperancinha/logistics/sensor/collector/configuration/DataCollectorConfiguration.java
package org.jesperancinha.logistics.sensor.collector.configuration;

import com.google.gson.Gson;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class DataCollectorConfiguration {

    @Bean
    Gson gson() {
        return new Gson();
    }

}

File: bl-central-server/bl-sensor-data-collector/src/main/kotlin/org/jesperancinha/logistics/sensor/collector/configuration/DataCollectorConfiguration.kt
package org.jesperancinha.logistics.sensor.collector.configuration

import com.fasterxml.jackson.databind.ObjectMapper
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class DataCollectorConfiguration {
    @Bean
    fun objectMapper(): ObjectMapper {
        return ObjectMapper()
    }
}